plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AbstractJdbcQueries.java

```diff
@@ -69,18 +82,22 @@ public AbstractJdbcQueries.MultiQueryOutput run(RunContext runContext) throws Ex
                     stmt.setFetchSize(runContext.render(this.getFetchSize()).as(Integer.class).orElseThrow());
                     logger.debug("Starting query: {}", query);
                     stmt.execute();
-                    if (!isTransactional) {
+                    if (!useTransactions && supportsTx) {
```

Comment:
Wait it's not logical because you check already `supportsTx` in `useTransactions`